js/google_ima.js

```diff
@@ -1818,6 +1838,16 @@ require("../html5-common/js/utils/utils.js");
           case eventType.USER_CLOSE:
           case eventType.SKIPPED:
           case eventType.COMPLETE:
+            //Workaround of an issue on iOS where the IMA iframe is capturing clicks after an ad.
+            //We will show the iframe on receiving STARTED and hide it when receiving COMPLETE
+            if (OO.isIos) {
+              var IMAiframe = $("iframe[src^='http://imasdk.googleapis.com/']")[0];
+              if (IMAiframe && IMAiframe.style)
+              {
+                IMAiframe.style.display = 'none';
```

Comment:
This is a slightly risky change, since we're directly modifying the style of an IMA iframe and IMA could release an update which breaks our integration with IMA. However, I could not find an alternative fix/workaround since the IMA iframe itself is capturing these clicks. Setting pointer-events none on the iframe parent does not work. The only thing that does seem to work is setting display none on the parent, but this prevents the ad from rendering under most circumstances.

Reply:
What does everyone think about this? If we don't put this workaround in, the user cannot interact with the skin outside of the control bar. This means that the giant pause or replay icons in the middle of the skin cannot be clicked if there was an IMA ad.

Reply:
Note that this PR is two bug fixes/workarounds in one. I could also split this change out into another PR if people don't agree with this change.

Reply:
Could we achieve something similar by tweaking the CSS of the plugin containers on the skin?

Reply:
That would be ideal, but as mentioned above, I cannot get anything to work unless I set display:none on the plugin containers, which will mess up IMA setup. One thing I could do is have the plugin container start with display:block for its initial hidden state, and after IMA is setup, switch between display:none and display:block depending on if the plugin container should be visible. This would increase complexity of the skin. But there is still no guarantee that IMA will be ok with this since they require the container to have a calculated size (with regards to the DOM) prior to setting up for an ad. What do you think?

Reply:
@aeng7 hmm maybe that approach could also work with a negative z-index instead of `display: none`? I'm not sure which approach I'd prefer though, it does sound a bit complicated.

Reply:
I've tried the negative z-index. It does not work. The iframe still captures clicks. It looks to be an iOS quirk because this is observed only on iOS